cluster/src/main/java/org/apache/iotdb/cluster/entity/Server.java

```diff
@@ -125,6 +131,14 @@ public void start() {
       Router.getInstance().showPhysicalNodes(groupId);
     }
 
+    try {
+      registerManager.register(ClusterMonitor.INSTANCE);
+    } catch (StartupException e) {
+      LOGGER.error("meet error while starting up.", e);
+      stop();
+      LOGGER.error("{} exit", IoTDBConstant.GLOBAL_DB_NAME);
```

Comment:
this log is useless because when iotdb stop, it will print this log

Reply:
deleted